helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml

```diff
@@ -14,7 +13,7 @@ spec:
           annotations:
             description: '{{`Scraping of all promtail pods to check if one failed every 5 minutes.`}}'
             opsrecipe: promtail-is-not-running/
-          expr: count(up{app="promtail"} == 0) > 0
+          expr: count(up{container="promtail"} == 0) > 0
```

Comment:
Is `cluster_type="management_cluster"` missing ?

Reply:
No promtail should alert on all clusters

Reply:
Even for customer promtail if they have one ?

Reply:
We do manage them so yes

Reply:
But it's mostly about those in kube-system